ui/css/module/_listitem.scss

```diff
@@ -10,4 +10,19 @@
   .list-item-text {
     @include paragraph-with-marker();
   }
+
+  .node-paragraph {
+    margin-bottom: 0;
+  }
```

Comment:
This feels a bit risky as a list item could contain multiple paragraphs. Is the goal to modify just the last one?

Reply:
Currently the list item has a `margin-bottom: 8px` based on the design. With the `.node-paragraph` inside, it is making the list item's `margin-bottom` closer to `24px`. The goal was to correct this. Given the multiple paragraph case, I'll make a fix.